test/e2e_node/services/kubelet.go

```diff
@@ -122,6 +140,34 @@ func (e *E2EServices) startKubelet() (*server, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	// set up the default kubeletconfiguration
```

Comment:
Add notice that if people want to add/remove option, they also need to change the flag below.
The logic duplication makes the code hard to maintain, but there doesn't seem to be a better solution.

Reply:
I will interleave the construction of the list of flag names with setting the field values, which should put the two steps as close together as possible. I will also add a comment.